src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -235,6 +235,24 @@
      * modifiers, type, the identifier name, and an optional
      * assignment statement.
      *
+     * <p>For example:</p>
+     * <pre>
+     * private int PI = 3.14;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --VARIABLE_DEF -&gt; VARIABLE_DEF
```

Comment:
please add `|` to each row as depicted in cmd output

Reply:
Since `VARIABLE_DEF` is te root node in this context, wouldn't the addition of `|` before it be somehwat redundant?

Reply:
I think no, i took as an example https://github.com/checkstyle/checkstyle/pull/9378/files

Reply:
Okay, I've made the changes